packages/web/src/components/nav/desktop/NowPlayingArtworkTile.tsx

```diff
@@ -53,7 +54,18 @@ const FadeInUp = (props: FadeInUpProps) => {
 
   return (
     <animated.div
-      className={styles.fadeIn}
+      css={{
+        border: '1px solid var(--currently-playing-border)',
```

Comment:
in general id like us to move away from using css variables, so if there is a way to define this in the theme, or use an existing theme value, that is ideal.

Reply:
Agreed. I don't know that this has a theme equivalent and I'm not sure design wants there to be one. Will investigate.

Reply:
Was able to update a lot of this to use harmony styles and variables via `Box`.